File: jvm/core/main/scala/com/webank/eggroll/core/resourcemanager/ResourceWrapper.scala
package com.webank.eggroll.core.resourcemanager

import java.util.concurrent.atomic.AtomicLong

case class ResourceWrapper(resourceType:String,
                     // allocate:AtomicLong=new AtomicLong(0),
                      total:AtomicLong=new AtomicLong(0),
                      used : AtomicLong=new AtomicLong(0),
                      allocated:AtomicLong=new AtomicLong(0)
                     ) ;

File: jvm/node_manager/src/main/java/org/fedai/eggroll/nodemanager/pojo/ResourceWrapper.java
package org.fedai.eggroll.nodemanager.pojo;

import lombok.Data;

import java.util.concurrent.atomic.AtomicLong;

@Data
public class ResourceWrapper {
    private String resourceType;
    AtomicLong total = new AtomicLong(0);
    AtomicLong used = new AtomicLong(0);
    AtomicLong allocated = new AtomicLong(0);

    public ResourceWrapper(String resourceType, AtomicLong total) {
        this.resourceType = resourceType;
        this.total = total;
    }
}
